python/searching/TernarySearch.py

```diff
@@ -0,0 +1,46 @@
+'''Ternary search is a decrease(by constant) and conquer algorithm that can be used to find an element in an array.
+It is same as the binary search. The only difference is that, it reduces the time complexity a bit more.
+Its time complexity is O(log n base 3) and that of binary search is O(log n base 2).'''
+
```

Comment:
Add a similar comment block in the other file as well

Reply:
> Add a similar comment block in the other file as well

Okay , Done👍🏽